Tests/Expected/XCAssets/swift4-context-all-customname.swift

```diff
@@ -0,0 +1,133 @@
+// Generated using SwiftGen, by O.Halligon — https://github.com/SwiftGen/SwiftGen
+
+#if os(OSX)
+  import AppKit.NSImage
+  typealias Color = NSColor
+  typealias Image = NSImage
+#elseif os(iOS) || os(tvOS) || os(watchOS)
+  import UIKit.UIImage
+  typealias Color = UIColor
+  typealias Image = UIImage
+#endif
+
+// swiftlint:disable file_length
+
+@available(*, deprecated, renamed: "XCTImageAsset")
+typealias XCTAssetsType = XCTImageAsset
+
+struct XCTImageAsset {
+  fileprivate var value: String
```

Comment:
Seeing as we decided to keep `value` for the context, should I still rename this one?